### DISCUSSION TITLE
Why are the results the same？fast = FALSE or TRUE in function “imbalanced”

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#187] [Q&A] [answered]
library(randomForestSRC)
data(breast, package = "randomForestSRC")
breast <- na.omit(breast)
f <- as.formula(status ~ .)
o.rfq1 <- imbalanced(f, breast, fast = FALSE, seed=9)
print(o.rfq1)
o.rfq2 <- imbalanced(f, breast, fast = TRUE, seed=9)
print(o.rfq2)
![image](https://user-images.githubusercontent.com/5112397/149727088-51330efb-51eb-4a80-a1f4-87894c9cd814.png)
![image](https://user-images.githubusercontent.com/5112397/149727188-9e07d422-5de0-4879-bc87-b5b2d4782623.png)

**Comment by ishwaran** [comment, marked as answer]
It just happens that the default setting for sampsize used by `rfsrc.fast`

`sampsize = function(x){min(x * .632, max(150, x ^ (3/4)))}`

gives exactly the same size as` rfsrc` for this data set.    Therefore they will give exactly the same answer.
